common/dispatcher/dispatcher.js

```diff
@@ -85,3 +85,94 @@ const showRequestHeaders = function(origin, uuid) {
 const cacheableShowRequestHeaders = function(origin, uuid) {
   return origin + dispatcher_path + `?uuid=${uuid}&cacheable&show-headers`;
 }
+
+// This script requires
+// - `/common/utils.js` for `token()`.
+
+// Represents an remote executor with ID `this.uuid`.
+// For more detailed explanation see `README.md`.
+class RemoteContext {
+  // Caller should create an executor with `this.uuid`.
```

Comment:
This doesn't bring new information, this comment just repeat the signature.
Maybe:
```
`uuid` is a UUID string that identifies the remote context and should match
 with the `uuid` parameter of the URL of the remote context.
```